crates/winmd/src/types/class.rs

```diff
@@ -125,7 +125,7 @@ impl Class {
             quote! {
                 #[repr(transparent)]
                 #[derive(Default, Clone, PartialEq)]
-                pub struct #name { ptr: ::winrt::ComPtr<#name> }
+                pub struct #name { ptr: ::winrt::ComPtr<#default_name> }
```

Comment:
This might be slightly controversial. Now classes are wrappers around ComPtrs to their default interfaces. This is of course functionaliy equivalent to what was there before, but means that the ABI type is different. For example, `windows::foundation::Uri`'s ABI type is now `RawComPtr<windows::foundation::IUriRuntimeClass>`. Again this is functionally equivalent, and it could be argued more "correct", but different than before.

Reply:
No objections, I would argue this is more correct.